src/gtk/widgets/recent_info.rs

```diff
@@ -67,15 +67,15 @@ impl RecentInfo {
     }
 
     pub fn get_added(&self) -> i64 {
-        unsafe { ffi::gtk_recent_info_get_added(GTK_RECENT_INFO(self.unwrap_widget())) }
+        unsafe { ffi::gtk_recent_info_get_added(GTK_RECENT_INFO(self.unwrap_widget())) as i64 }
```

Comment:
[`gtk_recent_info_get_added`](https://people.gnome.org/~shaunm/girdoc/C/Gtk.RecentInfo.get_added.html) returns a value of type `time_t`, Unix- and POSIX-compliant systems implement this as a signed integer.